detection-rules/link_google_translate.yml

```
@@ -10,7 +10,7 @@ authors:
 severity: "low"
 source: |
   type.inbound
-  and any(body.links, .href_url.domain.root_domain == "translate.goog")
+  and any(body.links, any(.href_url.rewrite.encoders, strings.contains(., "google_translate_open_redirect")))
```

We should still support old ones for backwards compatibility. Also we shouldn't be doing `strings.contains` on an enum and should only be doing `in` or `==`, otherwise you lose extra detection
```suggestion
  and any(body.links,
      .href_url.domain.root_domain == "translate.goog"
      or "google_translate_open_redirect" in .href_url.rewrite.encoders
  )
```

also FYI @hugh-sublime we don't have an editor diagnostic to help with this particular case